admin_guide/integrations.rst

```diff
@@ -0,0 +1,336 @@
+External Integrations
+#####################
+
+Ingesting Data from Kafka
+=========================
+
+Citus can leverage existing Postgres data ingestion tools. For instance, we can use a tool called `kafka-sink-pg-json <https://github.com/justonedb/kafka-sink-pg-json>`_ to copy JSON messages from a Kafka topic into a database table. As a demonstration, we'll create a ``kafka_test`` table and ingest data from the ``test`` topic with a custom mapping of JSON keys to table columns.
+
+The easiest way to experiment with Kafka is using the Confluent platform, which includes Kafka, Zookeeper, and associated tools whose versions are verified to work together.
+
+.. code-block:: bash
+
+  # we're using Confluent 2.0 for kafka-sink-pg-json support
+  curl -L http://packages.confluent.io/archive/2.0/confluent-2.0.0-2.11.7.tar.gz \
+    | tar zx
+
+  # Now get the jar and conf files for kafka-sink-pg-json
+  mkdir sink
+  curl -L https://github.com/justonedb/kafka-sink-pg-json/releases/download/v1.0.2/justone-jafka-sink-pg-json-1.0.zip \
```

Comment:
Hm, we must have different versions of `tar` -- mine could handle the zip. But I'll switch the instructions to use `unzip` since that's definitely meant for zip files.